docs/trex_debug_server.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Debugging Extensions in Tableau Server and Tableau Online 
+layout: docs
+---
+
+A dashboard extension embeds a web page inside of a dashboard on Tableau Server and Tableau Online. You can debug this embedded web browser using the debugging tools that are built into the browser that you are using.
+
+The following section describes how you could debug your extension using the [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/){:target="_blank"}. The same principles generally apply if you are using other browsers and their developer tools. 
+
+| For Tableau Desktop, see [Debugging Extensions in Tableau Desktop]({{site.baseurl}}/docs/trex_debugging.html).
+
+---
+**In this section**
+
+* TOC
+{:toc}
+
+
+
+---
+
+## Enable the debugging tools in the browser
+
+1. Open the workbook containing the extension to Tableau Server or Tableau Online.
```

Comment:
Yikes! Thanks, @johnDance.  I'll fix that. 
The  "Hello World" topic is in `docs\trex_create.md` .
The changes are not showing up here in the GitHub UI review tool because there are too many diffs.

Reply:
@johnDance  You can find the file for the "Hello World" topic if you click the **View Changes** button or click the **Files Changed** link in the pull request.  The file is collapsed (`/docs/trex_create.md`) because there are too many diffs, but you can click the **Load Diffs** link.